armonik/compute-plane.tf

```diff
@@ -442,6 +442,50 @@ resource "kubernetes_deployment" "compute_plane" {
             }
           }
         }
+
+        # Fluent-bit container windows
+        dynamic "container" {
+          for_each = (!var.fluent_bit.windows_is_daemonset ? [1] : [])
```

Comment:
Maybe you should add a condition on whether a windows node is needed

Reply:
That is already the case. The variable `var.fluent_bit.windows_is_daemonset` is defined only when we have a windows node.